ca.potatoe.cmdjewel.yml

```diff
@@ -0,0 +1,64 @@
+app-id: ca.potatoe.cmdjewel
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: cmdjewel
+finish-args:
+  - --socket=pulseaudio
+modules:
+  # Compiles libopenmpt.
+  # Not including libopenmpt was a blunder on my part--hopefully I can include it in later releases of cmdjewel. For now, this serves as a quick fix so cmdjewel can "just work"
+  # Keep in mind cmdjewel's music is on a good ol' Impulse Tracker file. Literally any version of OpenMPT will work. Probably even bass.dll.
+  # The source for this is https://github.com/flathub/org.srb2.SRB2Persona/blob/master/org.srb2.SRB2Persona.yaml.
+  - name: libopenmpt
+    config-opts:
+        - --disable-examples
+        - --disable-openmpt123
+        - --disable-static
+        - --disable-tests
+        - --without-portaudio
+        - --without-portaudiocpp
+    cleanup:
+        - /include
+        - /lib/*.la
+        - /lib/*.so
+        - /lib/pkgconfig
+        - /share/doc
+    sources:
+        - type: archive
+          url: https://lib.openmpt.org/files/libopenmpt/src/libopenmpt-0.7.13+release.autotools.tar.gz
+          sha256: dcd7cde4f9c498eb496c4556e1c1b81353e2a74747e8270a42565117ea42e1f1
+          x-checker-data:
+            type: anitya
+            project-id: 141468
+            stable-only: true
+            url-template: https://lib.openmpt.org/files/libopenmpt/src/libopenmpt-$version+release.autotools.tar.gz
+
+  # Loads the cmdjewel binary rather than fiddling with installing alsa and compiling it with Rust. This is much nicer. (Also I copied it from InfiniteShooter.)
+  # Possible TODO: fiddle with flatpak to compile new builds of cmdjewel from release.
+  - name: cmdjewel
+    buildsystem: simple
+    build-commands:
+      # cmdjewel binary (unzip)
+      - unzip linux.zip
+      - install -D linux/cmdjewel /app/bin/cmdjewel
+      # Metadata and a .desktop file
+      - install -D ca.potatoe.cmdjewel.desktop /app/share/applications/ca.potatoe.cmdjewel.desktop
+      - install -D ca.potatoe.cmdjewel.appdata.xml /app/share/metainfo/ca.potatoe.cmdjewel.appdata.xml
+      # Icons
+      - install -D icon-512x512.png /app/share/icons/hicolor/512x512/apps/ca.potatoe.cmdjewel.png
+      - install -D icon-256x256.png /app/share/icons/hicolor/256x256/apps/ca.potatoe.cmdjewel.png
+      - install -D icon-128x128.png /app/share/icons/hicolor/128x128/apps/ca.potatoe.cmdjewel.png
+      - install -D icon-64x64.png /app/share/icons/hicolor/64x64/apps/ca.potatoe.cmdjewel.png
+    sources:
+      # The game is pulled from GitHub
+      - type: file
```

Comment:
should be `type: archive`

Reply:
thanks for pointing that out! i actually just rewrote that section to build from source and use the appdata.xml, so i'm using `type: git` and `type: file` if that works